target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-wtr-m2133hp.dts

```diff
@@ -0,0 +1,420 @@
+// SPDX-License-Identifier: ISC
+/*
+ * Copyright (c) 2015 - 2016, The Linux Foundation. All rights reserved.
+ * Copyright (c) 2020 Yanase Yuki <dev@zpc.sakura.ne.jp>
+ */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "Buffalo WTR-M2133HP";
+	compatible = "buffalo,wtr-m2133hp", "qcom,ipq4019";
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x20000000>;
+	};
+
+	chosen {
+		/*
+		 * U-Boot adds "ubi.mtd=rootfs root=mtd:ubi_rootfs" to
+		 * kernel command line. But we use different partition names,
+		 * so we have to set correct parameters.
+		 */
+		bootargs-append = " ubi.mtd=ubi root=/dev/ubiblock0_1";
+	};
+
+	aliases {
+		led-boot = &power_blue;
+		led-failsafe = &power_orange;
+		led-running = &power_white;
+		led-upgrade = &power_blue;
+		label-mac-device = &gmac0;
+	};
+
+	soc {
+		rng@22000 {
+			status = "okay";
+		};
+
+		tcsr@1949000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1949000 0x100>;
+			qcom,wifi_glb_cfg = <TCSR_WIFI_GLB_CFG>;
+		};
+
+		tcsr@194b000 {
+			compatible = "qcom,tcsr";
+			reg = <0x194b000 0x100>;
+			qcom,usb-hsphy-mode-select = <TCSR_USB_HSPHY_HOST_MODE>;
+			status = "okay";
+		};
+
+		ess_tcsr@1953000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1953000 0x1000>;
+			qcom,ess-interface-select = <TCSR_ESS_PSGMII>;
+		};
+
+		tcsr@1957000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1957000 0x100>;
+			qcom,wifi_noc_memtype_m0_m2 = <TCSR_WIFI_NOC_MEMTYPE_M0_M2>;
+		};
+
+		crypto@8e3a000 {
+			status = "okay";
+		};
+
+		watchdog@b017000 {
+			status = "okay";
+		};
+
+		ess-switch@c000000 {
+			status = "okay";
+			switch_lan_bmp = <0x1c>;
+		};
+
+		ess-psgmii@98000 {
+			status = "okay";
+		};
+
+		edma@c080000 {
+			status = "okay";
+		};
+
+		usb3@8af8800 {
+			status = "okay";
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power_white: power_white {
```

Comment:
Please use "led_" prefix for the DT labels, as those are used in global context. E.g.
`led_power_white: power_white {`

Reply:
fixed.